File: robot/java/RobotContainer.java
package frc.robot;

import edu.wpi.first.wpilibj2.command.Commands;
import frc.robot.subsystems.VoiceControl;

/** Example container. Keep yours; you only need the VoiceControl lines. */
public class RobotContainer {
  private final VoiceControl voice = new VoiceControl();

  public RobotContainer() {
    configureVoiceCommands();
  }

  private void configureVoiceCommands() {
    // Replace these with your real commands.
    voice.bindCommand("INTAKE", Commands.print("INTAKE"));
    voice.bindCommand("SHOOT", Commands.print("SHOOT"));
    voice.bindCommand("CLIMB", Commands.print("CLIMB"));
    // "STOP" is pre-bound to cancelAll(); override with voice.bind("STOP", ...) if needed.
  }
}

File: robot/cpp/RobotContainer.cpp
#include "RobotContainer.h"

#include <frc2/command/Commands.h>

RobotContainer::RobotContainer() {
  ConfigureVoiceCommands();
}

void RobotContainer::ConfigureVoiceCommands() {
  // Replace these with your real commands.
  m_voice.BindCommand("INTAKE", frc2::cmd::Print("INTAKE"));
  m_voice.BindCommand("SHOOT", frc2::cmd::Print("SHOOT"));
  m_voice.BindCommand("CLIMB", frc2::cmd::Print("CLIMB"));
  // "STOP" is pre-bound to CancelAll(); override with m_voice.Bind("STOP", ...) if needed.
}
